### ISSUE TITLE
Died - after 7.5 hours 

### ISSUE THREAD

**Issue by pgaston** [keras-team/autokeras#1541]
### Bug Reproduction
Code for reproducing the bug:
reg = ak.ImageRegressor(overwrite=True, 
                        max_trials=150,      # 100 is default - first 15 take roughly an hour
                        metrics=['mae'],    # mean average error is more useful, though as a loss need mse
                        loss='mse',
                        project_name="GMA pallet pose estimation",
                        directory="autoK",
                        seed=42)
logdir = "aklogs"
tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=logdir, histogram_freq=1)

removeFilesFromFolder("./aklogs")   # clean up for new tensorboard

# Feed the image regressor with training data.
reg.fit(x_train, y_train, 
        validation_data=(x_val, y_val),
        callbacks=[tensorboard_callback])
Data used by the code:
X = 250M of images, Y = 3 floats per image
### Setup Details
Include the details about the versions of:
 - OS type and version: ubuntu 20.04
 - Python: 3.9.2
 - autokeras: 
 - keras-tuner:  whatever the current install is
 - scikit-learn:
 - numpy:
 - pandas:   not used
 - tensorflow:   2.4.1

### Additional context

Just wondering if it makes sense to keep trying this out...    My loss wasn't getting better for over 4 hours.

Thanks!
p
